### ISSUE TITLE
--version argument to include the vsc version

### ISSUE THREAD

**Issue by JtMotoX** [coder/code-server#4163]
Current:
```bash
code-server --version
3.11.1 acb5de66b71a3f8f1cc065df4633ecd3f9788d46
```
Desired (example):
```bash
code-server --version
3.11.1 acb5de66b71a3f8f1cc065df4633ecd3f9788d46 vsc-1.57.1
```

**Comment by jsjoeio** [comment]
Neat idea! Thanks for raising. Added to the Backlog Candidates which just means we'll wait for the community to weigh in.

**Comment by TeffenEllis** [comment]
We technically support this via `code-server —version —json`

```sh
$ code-server --version --json
{
  codeServer: '3.11.1',
  commit: 'b9d79dc1988a30e129aa6452289cc69ad5da7a6c',
  vscode: '1.60.0'
}
```

**Comment by JtMotoX** [comment]
> We technically support this via `code-server —version —json`

Oh ok I didn't realize there was a json flag.  Maybe I should have read the documnetation.  This does not really solve my need though since this is not a valid json response.  In the current response, json parsers cannot read it because it is not valid json.  Json should have double quotes around the keys and values.  Right now there is no double quotes around the keys, and the values have single quotes instead of double.

```bash
> code-server --version --json | jq '.vscode'
parse error: Invalid numeric literal at line 2, column 13
```

![image](https://user-images.githubusercontent.com/7191259/133496789-1cde702d-e676-4ae2-8faf-a1a850ee8c8a.png)

![image](https://user-images.githubusercontent.com/7191259/133496930-af907a7a-ba98-4009-9e50-bd03d5dfd193.png)

**Comment by TeffenEllis** [comment]
@JtMotoX you are absolutely correct, and that’s a very easy fix. We can likely have it out in 3.12.1, however, here’s a temp fix:

```sh
$ node -e "console.log(JSON.stringify($(code-server --version --json)))”
{"codeServer":"3.11.1","commit":"acb5de66b71a3f8f1cc065df4633ecd3f9788d46","vscode":"1.57.1"}
```

**Comment by JtMotoX** [comment]
This workaround works.  I will use this until the changes are applied on your side.  Thanks 👍

```bash
> node -e "console.log(JSON.stringify($(code-server --version --json)))" | jq -r '.vscode'; code --version | head -n1
1.57.1
1.57.1
```

**Comment by jsjoeio** [comment]
This was done in https://github.com/coder/code-server/pull/4925 and is included in 4.1.0

**Comment by jsjoeio** [comment]
Duplicate of https://github.com/coder/code-server/issues/4874
